frame/conviction-voting/src/lib.rs

```diff
@@ -0,0 +1,625 @@
+			Ok(())
+		}
+
+		/// Remove a vote for a poll.
+		///
+		/// If:
+		/// - the poll was cancelled, or
+		/// - the poll is ongoing, or
+		/// - the poll has ended such that
+		///   - the vote of the account was in opposition to the result; or
+		///   - there was no conviction to the account's vote; or
+		///   - the account made a split vote
+		/// ...then the vote is removed cleanly and a following call to `unlock` may result in more
+		/// funds being available.
+		///
+		/// If, however, the poll has ended and:
+		/// - it finished corresponding to the vote of the account, and
+		/// - the account made a standard vote with conviction, and
+		/// - the lock period of the conviction is not over
+		/// ...then the lock will be aggregated into the overall account's lock, which may involve
+		/// *overlocking* (where the two locks are combined into a single lock that is the maximum
+		/// of both the amount locked and the time is it locked for).
+		///
+		/// The dispatch origin of this call must be _Signed_, and the signer must have a vote
+		/// registered for poll `index`.
+		///
+		/// - `index`: The index of poll of the vote to be removed.
+		/// - `class`: Optional parameter, if given it indicates the class of the poll. For polls
+		///   which have finished or are cancelled, this must be `Some`.
+		///
+		/// Weight: `O(R + log R)` where R is the number of polls that `target` has voted on.
+		///   Weight is calculated for the maximum number of vote.
+		#[pallet::weight(T::WeightInfo::remove_vote())]
+		pub fn remove_vote(
+			origin: OriginFor<T>,
+			class: Option<ClassOf<T>>,
+			index: PollIndexOf<T>,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			Self::try_remove_vote(&who, index, class, UnvoteScope::Any)
+		}
+
+		/// Remove a vote for a poll.
+		///
+		/// If the `target` is equal to the signer, then this function is exactly equivalent to
+		/// `remove_vote`. If not equal to the signer, then the vote must have expired,
+		/// either because the poll was cancelled, because the voter lost the poll or
+		/// because the conviction period is over.
+		///
+		/// The dispatch origin of this call must be _Signed_.
+		///
+		/// - `target`: The account of the vote to be removed; this account must have voted for poll
+		///   `index`.
+		/// - `index`: The index of poll of the vote to be removed.
+		/// - `class`: The class of the poll.
+		///
+		/// Weight: `O(R + log R)` where R is the number of polls that `target` has voted on.
+		///   Weight is calculated for the maximum number of vote.
+		#[pallet::weight(T::WeightInfo::remove_other_vote())]
+		pub fn remove_other_vote(
+			origin: OriginFor<T>,
+			target: T::AccountId,
+			class: ClassOf<T>,
+			index: PollIndexOf<T>,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let scope = if target == who { UnvoteScope::Any } else { UnvoteScope::OnlyExpired };
+			Self::try_remove_vote(&target, index, Some(class), scope)?;
+			Ok(())
+		}
+	}
```

Comment:
I think we should make an extrinsic which allows any user to reduce or remove another users vote if their free balance falls below their vote balance. For example if a user get slashed, their vote amount will not get updated automatically, but such a call would provide a mechanism for the community to rebalance this.

Reply:
https://github.com/paritytech/substrate/issues/10761